src/Core/Prefab/GlobeView.js

```diff
@@ -51,17 +50,17 @@ import { ellipsoidSizes } from 'Core/Math/Ellipsoid';
 
 /**
  * Globe's EVENT
- * @property GLOBE_INITIALIZED {string} emit one time when globe is initialized
- * @property LAYER_ADDED {string} emit when layer id added in viewer
- * @property LAYER_REMOVED {string} emit when layer id removed in viewer
- * @property COLOR_LAYERS_ORDER_CHANGED {string} emit when  color layers order change
+ * @property GLOBE_INITIALIZED {string} Deprecated: emit one time when globe is initialized (use VIEW_EVENTS.INITIALIZED instead).
```

Comment:
Add a version when this will be removed to not forget about it (let's say 2.13 ou 2.14)

Reply:
done